package.json

```diff
@@ -43,7 +42,7 @@
     "eslint-plugin-vue": "^5.2.3",
     "sass": "^1.25.0",
     "sass-loader": "^8.0.0",
-    "typescript": "<3.6.0",
+    "typescript": "3.7.3",
```

Comment:
We didn't want to go past 3.6.0 as that broke other package dependencies. I note that you aren't updating other packages so something is probably broken somewhere or at least you're getting npm or build warnings/errors.

There's another ticket to upgrade TS because it's not as straightforward as this unfortunately.

I will try to help with the packages on Monday.

Reply:
If you rebase to the latest master, the Typescript package (and many others) have been updated, so you should not have to change this line.